pom.xml

```diff
@@ -83,6 +83,7 @@
         <commons-cli.version>1.5.0</commons-cli.version>
         <commons-io.version>2.11.0</commons-io.version>
         <json-simple.version>1.1.1</json-simple.version>
+        <slf4j.version>2.0.6</slf4j.version>
```

Comment:
Do we need this specific version? We import `spring-boot-dependencies` using **1.7.36**

Reply:
I will remove the version and switch to logback due to tests.